torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4024,6 +4024,21 @@ def _tensor(shape, dtype=dtype, low=None, high=None):
 
     return [SampleInput(tensor, args=args) for tensor, args in test_cases]
 
+def sample_inputs_attn(op_info, device, dtype, requires_grad, **kwargs):
+    def _tensor(shape):
+        return make_tensor(shape, device, dtype, low=None, high=None, requires_grad=requires_grad)
+
+    def generator():
+        for dim1 in range(1, 10):
```

Comment:
I'm not sure doing an exhaustive loop over all possible size is the best here to keep test time low :D 
Maybe test a few cases only?